src/utils/actions.js

```diff
@@ -0,0 +1,325 @@
+// this file provides functions for various action codes
+// it is assumed every function recieves a "context" object
+// Context:
+//  Patient: identifier for the patient whose data triggered the workflow
+//  records: list of records
+//  Encounter: encounter where some data triggered the workflow
+//  Organization: Organization that is doing the workflow
+//  PlanDef: *** do we want to keep a "pre-processed" version of the plandef?
+//  Action: action currently in context from the planDef
+//  client: set of clients to connect to
+//      source: FHIR client to make additional queries to EHR
+//      dest: FHIR client to send to PHA/TTP
+//      trust services
```

Comment:
We should sync up on these. It looks like your `.submit` is assuming the entire axios object (with the `result.status` ) property, but the data trust service requests are expecting just the data which comes from `result.data`). 
My current implementation of these functions is to just return the data, but since you use other parts and handle errors here I'm leaning towards all of these networking calls to just return the axios promise. That means for this PR `result` needs to be updated to `result.data` on
- [ ] line 102
- [ ] line 119
- [ ] line 136